public/app/plugins/datasource/azuremonitor/azure_log_analytics/azure_log_analytics_datasource.ts

```diff
@@ -252,4 +252,16 @@ export default class AzureLogAnalyticsDatasource extends DataSourceWithBackend<
   async getAzureLogAnalyticsCheatsheetQueries() {
     return await this.getResource(`${this.resourcePath}/v1/metadata`);
   }
+
+  async getBasicLogsQueryUsage(query: AzureMonitorQuery, table: string): Promise<number> {
+    const templateSrv = getTemplateSrv();
+    const data = {
+      table: table,
+      resource: query.azureLogAnalytics?.resources?.[0],
```

Comment:
Maybe we should run the replace on this property as well incase the resource is specified as a template variable.

Reply:
Great catch, thanks for pointing it out. I added this change and also ended up finding out that the toggle was not properly shown for variables either, so I added this change as well and some tests to cover that :)